src/Appl/BeamBunch.f90

```diff
@@ -400,6 +405,7 @@ subroutine drift1_BeamBunch(this,beamln,z,tau,bitype,nseg,nst,ihlf)
                                                 this%Pts1(6,i)
             enddo
 
+            print*,"biaobin is here."
```

Comment:
See Ji's comment:
```suggestion
```

Reply:
The above fix is added in the new version now.

Reply:
The above fix is added in the new version now.